hack/jenkins/e2e.sh

```diff
@@ -593,6 +593,24 @@ case ${JOB_NAME} in
     TEST_CLUSTER_RESYNC_PERIOD="--min-resync-period=12h"
     ;;
 
+  # Runs e2e on GCE with flannel and VXLAN.
+  kubernetes-e2e-gce-flannel)
+    : ${E2E_CLUSTER_NAME:="jenkins-gce-e2e-flannel"}
+    : ${E2E_PUBLISH_GREEN_VERSION:="true"}
+    : ${E2E_NETWORK:="e2e-gce-flannel"}
+    : ${GINKGO_TEST_ARGS:="--ginkgo.skip=$(join_regex_allow_empty \
```

Comment:
Is there any reason, why we don't want to run the same tests as we are running in e2e-gce ?